utils/validators.ts

```diff
@@ -513,10 +515,11 @@ export const parseJoinOrganizationPOSTRequest = function (params: JoinOrganizati
 
 export const parseUpdateOrganizationPUTRequest = function (params: UpdateOrganizationPUTParameters): UpdateOrganizationInput {
   if (params.userId === '' || params.userId === undefined) throw new Error(RESPONSE_MESSAGES.USER_ID_NOT_PROVIDED_400.message)
-  if (params.name === '' || params.name === undefined) throw new Error(RESPONSE_MESSAGES.ORGANIZATION_NAME_NOT_PROVIDED_400.message)
+  if ((params.name === '' || params.name === undefined) && (params.address === '' || params.address === undefined)) throw new Error(RESPONSE_MESSAGES.ORGANIZATION_MISSING_PARAMS_400.message)
```

Comment:
Just a nitpick, but this error name "ORGANIZATION_MISSING_PARAMS_400" leads one to believe  that the organization itself is missing these, better if it was something like `MISSING_PARAMS_TO_UPDATE_ORGANIZATION_400`